src/main/java/uk/gov/justice/hmpps/casenotes/controllers/ControllerAdvice.java

```diff
@@ -51,6 +52,17 @@
                         .build());
     }
 
+    @ExceptionHandler(ValidationException.class)
+    public ResponseEntity<ErrorResponse> handleException(final ValidationException e) {
+        log.debug("Bad Request (400) returned", e);
+        return ResponseEntity
+                .status(HttpStatus.BAD_REQUEST)
+                .body(ErrorResponse
+                        .builder()
+                        .status(HttpStatus.BAD_REQUEST.value())
+                        .build());
```

Comment:
not including any error messsage in the response?

Reply:
although will be confusing as we already catch `Exception` and return `BAD_REQUEST` with a developer message so would be worth putting one in...

Reply:
Catching exception and returning bad request looks a bit wrong - shouldn't a generic error return internal server error?

Reply:
Agreed!